src/app/public/modules/datepicker/datepicker.component.scss

```diff
@@ -11,3 +11,7 @@
 .sky-datepicker-calendar-container {
   position: fixed;
 }
+
+.sky-datepicker-hidden {
+  opacity: 0;
```

Comment:
I realize other implementations use width/height/overlay, but this `opacity` approach was to allow the affix service to properly position the affixed element before displaying it. I think that may be in addition to what we're trying to achieve with the former. Maybe we should chat about this so I can understand how best to do both.

Reply:
You might want to use `visibility: hidden` instead, since with opacity, the date picker is still interactable. To test, open the picker then scroll the window until the picker is hidden. You can still click on the calendar controls.